apps/css-workshop/pages/index.html

```diff
@@ -141,25 +131,25 @@ <h1>iTwinUI-CSS</h1>
     <img
       alt="GitHub last commit"
       src="https://img.shields.io/github/last-commit/itwin/iTwinUI"
+      draggable="false"
     >
     <a
       href="https://www.npmjs.com/package/@itwin/itwinui-css"
       rel="nofollow"
+      style="text-decoration: none;"
     >
       <img
-        src="https://camo.githubusercontent.com/d372d1dae45885e50fb718a9c0a1a20544a3b0205842ca6cc4447545249c0e73/68747470733a2f2f696d672e736869656c64732e696f2f6e706d2f762f40697477696e2f697477696e75692d637373"
+        src="https://badge.fury.io/js/@itwin%2Fitwinui-css.svg"
         alt="itwinui-css on npm"
+        draggable="false"
         data-canonical-src="https://img.shields.io/npm/v/@itwin/itwinui-css"
-        style="max-width: 100%;"
       >
     </a>
-    <a
-      href="https://itwin.github.io/iTwinUI-react/"
-      target="_blank"
-    >
+    <a href="https://itwin.github.io/iTwinUI/react/">
```

Comment:
Imo we should remove all badges, because this site is not meant to public facing. In the past we didn't have a documentation site but now we do, so there is no purpose to having this fluff on here

Reply:
Removed badges and GitHub corner banner.